snuba/snuba_migrations/metrics_summaries/0001_metrics_summaries_create_table.py

```diff
@@ -0,0 +1,86 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime, Nested, String, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+from snuba.utils.schemas import Float
+
+storage_set_name = StorageSetKey.METRICS_SUMMARIES
+local_table_name = "metrics_summaries_local"
+dist_table_name = "metrics_summaries_dist"
+
+UNKNOWN_SPAN_STATUS = 2
+
+columns: List[Column[Modifiers]] = [
+    # ids
+    Column("project_id", UInt(64)),
+    Column("trace_id", UUID()),
+    Column("span_id", UInt(64)),
+    Column("segment_id", UInt(64)),
+    # metrics summary
+    Column("min", Float(64)),
+    Column("max", Float(64)),
+    Column("sum", Float(64)),
+    Column("count", Float(64)),
+    # metrics metadata
+    Column("tags", Nested([("key", String()), ("value", String())])),
+    # span metadata
+    Column("start_timestamp", DateTime()),
+    Column("exclusive_time", Float(64)),
+    Column("op", String(Modifiers(low_cardinality=True))),
+    Column("group", UInt(64)),
```

Comment:
What is `group` in this context?

Reply:
`group` is one way to select span examples. We might want to target exact spans producing those specific metric values or spans looking like them.

We might be able to not have that and just focus on returning span IDs.